README.md

```diff
@@ -8,8 +8,16 @@ You can find the complete guidance on publishing personal identifiers in the [Be
 
 On the other hand, if BO information is collected in a central register via other means, then it is recommended to publish that information as a separate dataset. The BO dataset and the OCDS dataset should [identify organizations](https://standard.open-contracting.org/latest/en/schema/identifiers/#organization-ids) in the same way, so that users can cross-reference the datasets. The BO dataset can follow the [Beneficial Ownership Data Standard](https://standard.openownership.org/en/latest/).
 
+## Legal context
+
+In the European Union, this extension's fields correspond to [eForms BG-703 (Organisation) and BG-4 (Winner)](https://github.com/eForms/eForms). See [OCDS for eForms](https://standard.open-contracting.org/profiles/eforms/latest/en/) for the correspondences to eForms fields.
```

Comment:
The OCDS for eForms site doesn't actually mention the BG's at all so this instruction doesn't quite work. Suggest rather than linking to the eForms github instead link to https://docs.ted.europa.eu/eforms/latest/reference/business-terms/index.html where the user can look up which terms are in each group and then they can use the OCDS for eForms mapping.

"In the European Union, this extension's fields correspond to [eForms BG-703 (Organisation) and BG-4 (Winner)(https://docs.ted.europa.eu/eforms/latest/reference/business-terms/index.html). See [OCDS for eForms](https://standard.open-contracting.org/profiles/eforms/latest/en/) for the correspondences to eForms fields."

Reply:
Good spot. Updated in https://github.com/open-contracting-extensions/ocds_beneficialOwners_extension/pull/2/commits/c7b33378e740b077356e9082ab7c07d487dff2c6. @jpmckinney should we update this link in all the extensions that refer to the eForms GitHub repo?

Reply:
I've updated the links now.

Reply:
Thanks!